app/assets/stylesheets/sulBase.css

```diff
@@ -91,6 +91,21 @@ label.toggle-bookmark {
 .btn-outline-secondary.remove:hover svg {
   fill: var(--stanford-digital-red);
 }
+
+.btn-outline-danger {
```

Comment:
Should this be in the component library?

Reply:
These are now btn-outline-secondary, which are already defined in the component library